code/__DEFINES/sound.dm

```diff
@@ -1,3 +1,4 @@
 //max channel is 1024. Only go lower from here, because byond tends to pick the first availiable channel to play sounds on
 #define CHANNEL_LOBBYMUSIC 1024
 #define CHANNEL_ADMIN 1023
+#define CHANNEL_VOX 1022
```

Comment:
Why rely on a proc to randomly pick the right channel, when having it set to `0` will always ensure the sound is played on an open channel, natively? Seems like needless proc call overhead...not to mention, you'll end up relying on chance, at times, that they don't overlap.